src/pages/fileCentricCart/cartController.js

```diff
@@ -1,21 +1,34 @@
 import React from 'react';
+// import CircularProgress from '@material-ui/core/CircularProgress';
 import { useQuery } from '@apollo/client';
-import { getCart } from './store/cart';
+import { getCart, updateSortOrder } from './store/cart';
 import { Typography } from '../../components/Wrappers/Wrappers';
 import { GET_MY_CART_DATA_QUERY, GET_MY_CART_DATA_QUERY_DESC, table } from '../../bento/fileCentricCartWorkflowData';
 import CartView from './cartView';
 
 const cartController = () => {
   const cart = getCart();
   const ids = cart.fileIds ? cart.fileIds : [];
-  const defaultSortDirection = table.defaultSortDirection || 'asc';
+  let sortColumnValue = cart.sortColumn;
```

Comment:
Can you add the conditional ternary if its empty use the default sort from the bento configuration of cart

Reply:
I have already added the conditional ternary if it's empty use the default sort from the bento configuration of the cart, it is in the "if" statement line 14 of the cartControler.js.

Reply:
But I think the old statement is not useful anymore, let me add the update line inside that if statement.